src/main/java/betterquesting/api/enums/EnumFrameType.java

```diff
@@ -0,0 +1,40 @@
+package betterquesting.api.enums;
+
+public enum EnumFrameType {
+    DEFAULT,
+    SQUARE,
+    ROUNDED_SQUARE,
+    CIRCLE,
+    HEXAGON,
+    HEXAGON90,
+    PENTAGON,
+    DIAMOND,
+    STAR4,
+    CROSS,
+    DIAGONAL_CROSS,
+    DIAGONAL_STAR4,
+    PRICKLE,
+    GEAR,
+    GATE,
```

Comment:
MAIN as a prop should be deprecated, and should be converted into this frame option